app/utils/config.example.js

```diff
@@ -0,0 +1,4 @@
+
+module.exports = {
+  GOOGLE_API_KEY : 'REPLACE_ME'
```

Comment:
We still need a way to configure the key at deploy time, which means serving an additional script with the config or find-replace on `bundle.js`. That's not a high priority for me yet, I think, unless staging won't work without it.